cumulusci/tasks/robotframework/tests/test_robotframework.py

```diff
@@ -458,6 +459,101 @@ def test_pageobject(self):
         assert "created {}".format(output) in self.task_log["info"]
         assert os.path.exists(output)
 
+    def test_csv(self):
+        path = ",".join(
+            (
+                os.path.join(self.datadir, "TestLibrary.py"),
+                os.path.join(self.datadir, "TestResource.robot"),
+                os.path.join(self.datadir, "TestPageObjects.py"),
+            )
+        )
+        output = os.path.join(self.tmpdir, "keywords.csv")
```

Comment:
I think we are moving towards Pathlib.

Reply:
Good point. I modified this new code, but I'm not going to refactor the whole test in the interest of getting this finished this sprint.

Reply:
It's a minor issue but you can replace many of your uses of os.path with pathobj.foo() but it's up to you whether you change this code today.